src/scenes/ContactUs/index.tsx

```diff
@@ -41,7 +41,7 @@ export default function ContactUsView(): JSX.Element {
         <i>"{process.env.REACT_APP_TERRAWARE_FE_BUILD_VERSION || 'n/a'}"</i>
       ) as string,
       buttonText: strings.REPORT_PROBLEM,
-      link: docLinks.report_a_problem,
+      link: `${docLinks.report_a_problem}?build=${process.env.REACT_APP_TERRAWARE_FE_BUILD_VERSION || ''}`,
```

Comment:
There's an app version selector in redux, it might be cleaner to leverage that.
See L#26 in [DetectAppVersion](https://github.com/terraware/terraware-web/blob/main/src/components/common/DetectAppVersion.tsx#L26).
Something similar but using the [app version selecto](https://github.com/terraware/terraware-web/blob/main/src/components/common/DetectAppVersion.tsx#L26)r instead (this was Ryder's initial setup help).